rpcserver.go

```diff
@@ -2088,33 +2084,26 @@ func handleGetBlock(s *rpcServer, cmd interface{},
 		return hex.EncodeToString(blkBytes), nil
 	}
 
-	// The verbose flag is set, so generate the JSON object and return it.
-	blockInMainChain, _ := s.chain.MainChainHasBlock(hash)
-	var idx int64
-	if !blockInMainChain {
-		idx = -1
-	}
+	best := s.chain.BestSnapshot()
+
+	// See if this block is an orphan and adjust Confirmations accordingly.
+	onMainChain, _ := s.chain.MainChainHasBlock(hash)
```

Comment:
The snapshot only contains a few key pieces of data such as the most recent block height, hash, size, number of transactions, etc.  It's not a full chain snapshot.